pkg/csi/service/wcp/controller.go

```diff
@@ -710,12 +710,15 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		CryptoKeyID:             cryptoKeyID,
 	}
 
+	volFromSnapshotOnTargetDs := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
+		common.VolFromSnapshotOnTargetDs)
 	createVolumeOpts := common.CreateBlockVolumeOptions{
 		FilterSuspendedDatastores:      filterSuspendedDatastores,
 		UseSupervisorId:                isTKGSHAEnabled,
 		IsVdppOnStretchedSvFssEnabled:  isVdppOnStretchedSVEnabled,
 		IsByokEnabled:                  isByokEnabled,
 		IsCSITransactionSupportEnabled: isCSITransactionSupportEnabled,
+		VolFromSnapshotOnTargetDs:      volFromSnapshotOnTargetDs,
```

Comment:
Is this only wcp feature or vanilla as well?

Reply:
I only added support for wcp for now as that's where the requirements are coming from, but maybe I should add it for vanilla as well.